### PR TITLE
Only update what is needed.

### PR DESCRIPTION
**Pull request by almathew**
## What's this PR do?
Diffs updates against jobs already in Redis, in hopes that we're only updating what's needed.

### CODE REVIEW

File: import_jobs/utils.py [hunk 1]
```diff
@@ -222,3 +223,63 @@ def update_business_unit_modified_dates(business_unit, crawled_date,
     if updated:
         business_unit.date_updated = datetime.datetime.utcnow()
     return business_unit
+
+
+def should_update(jobs):
```
Comment (mklauber) [inline import_jobs/utils.py]: I feel like this is basically a pointless method, given all the work is being done in diff_jobs.

Comment (almathew) [inline import_jobs/utils.py]: I don't disagree, but it was an easy "this does exactly what I need it to" access method.

Comment (mklauber) [inline import_jobs/utils.py]: Is the return of `diff_jobs` a tuple for testing purposes?

Comment (almathew) [inline import_jobs/utils.py]: ....I get your point. Fine. I'll change it.

Comment (almathew) [inline import_jobs/utils.py]: Your opinions have been accounted for.

**Pull request merged**